config/locales/en.yml

```diff
@@ -1113,6 +1113,11 @@ en:
         invalid_event: 'cannot transition when %{state}'
         invalid_transition: 'cannot transition via "%{event}"'
         not_path_format: 'must be a path separated by "/". E.g. "" or "my/path"'
+      template:
+        header:
+          one: "1 error prohibited this %{model} from being saved"
+          other: "%{count} errors prohibited this %{model} from being saved"
+        body: "There were problems with the following fields:"
```

Comment:
I assume the `dynamic_form` version did not use our internal override so we didn't need these entries with it. And now we do. Correct?